app/models/event.rb

```diff
@@ -4,7 +4,7 @@ class Event < ApplicationRecord
 
   validates :name, presence: true
   validates :start_time, presence: true
-  validates :start_time, presence: true
+  validates :end_time, presence: true
```

Comment:
Events need a start time, but don't necessarily need an end time - think reminders for due dates & that kind of thing that are more instantaneous than an actual event with a timespan attached.